upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,18 @@
 == Migration Changes
 
+=== Migrating to 11.0.0
+
+==== LDAP no-import bugfix
+
+In the previous {project_name} version, when LDAP provider was configured with `Import Users` OFF, then it was possible to update the
```

Comment:
Add "the" before LDAP

Remove the "then"  It adds no meaning.

Reply:
Will change both things as you suggested.